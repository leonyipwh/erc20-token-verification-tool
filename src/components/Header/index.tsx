import AppBar from '@mui/material/AppBar';
import Box from '@mui/material/Box';
import Toolbar from '@mui/material/Toolbar';
import Button from '@mui/material/Button';
// import Logo from '../Logo';

const Header = () => {
  const pages: string[] = ['Token Verification'];

  return (
    <AppBar position="static">
      <Toolbar>
        {/* <Logo /> */}
        <Box sx={{ flexGrow: 1 }}>
          {pages.map((page) => (
            <Button
              key={page}
              sx={{ my: 2, color: 'white', display: 'block' }}
            >
              {page}
            </Button>
          ))}
        </Box>
      </Toolbar>
    </AppBar>
  );
};

export default Header;
