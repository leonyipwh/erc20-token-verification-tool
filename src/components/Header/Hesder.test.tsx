import { render } from '@testing-library/react';
import Header from './index';

describe('<Header>', () => {
  it('should render', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
