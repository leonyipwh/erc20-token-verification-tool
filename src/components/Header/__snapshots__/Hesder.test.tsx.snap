// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header> should render 1`] = `
<DocumentFragment>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
  >
    <div
      class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
    >
      <div
        class="MuiBox-root css-i9gxme"
      >
        <button
          class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root  css-1q39md6-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="button"
        >
          Token Verification
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </header>
</DocumentFragment>
`;
